docs/reference/connector/apis/create-connector-sync-job-api.asciidoc

```diff
@@ -0,0 +1,80 @@
+[[create-connector-sync-job-api]]
+=== Create connector sync job API
+++++
+<titleabbrev>Create connector sync job</titleabbrev>
+++++
+
+Creates a connector sync job.
+
+////
+[source, console]
+----
+PUT _connector/connector-id
+{
+  "index_name": "search-google-drive",
+  "name": "My Connector",
+  "service_type": "google_drive"
+}
+----
+// TESTSETUP
+////
+[source, console]
+--------------------------------------------------
+POST _connector/_sync_job
+{
+  "id": "connector-id",
+  "job_type": "full",
+  "trigger_method": "on_demand"
+}
+--------------------------------------------------
```

Comment:
We should consider cleaning up the connector and sync job that are created in this snippet, similar to what's done [here](https://github.com/elastic/elasticsearch/blob/main/docs/reference/connector/apis/create-connector-api.asciidoc?plain=1#L22). Otherwise, it may lead to test errors down the line.

Reply:
As discussed via Slack we'll skip this step as there's currently no way of using the generated id inside the URL path